packages/terra-clinical-alert/src/Alert.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IconRequired from 'terra-icon/lib/icon/IconRequired';
+import IconDiamond from 'terra-icon/lib/icon/IconDiamond';
+import BaseAlert from 'terra-alert';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Alert.scss';
+
+const cx = classNames.bind(styles);
+
+const AlertTypes = Object.assign({
+  GAP_CHECKING: 'gapChecking',
+  OUTSIDE_RECORDS: 'outsideRecords',
+}, BaseAlert.Opts.Types);
+
+const propTypes = {
+  /**
+   * An action element to be added to the action section of the alert.
+   */
+  action: PropTypes.element,
+  /**
+   * Child Nodes providing the message content for the alert. Can contain text and HTML.
+   */
+  children: PropTypes.node,
+  /**
+   * The icon to be used for an alert of type custom.
+   */
+  customIcon: PropTypes.element,
+  /**
+   * The status bar color to be used for an alert of type custom.
+   */
+  customStatusColor: PropTypes.string,
+  /**
+   * Callback function triggered when Dismiss button is clicked. The presence of this prop will cause the Dismiss button to be included on the alert.
+   */
+  onDismiss: PropTypes.func,
+  /**
+   * The title for the alert which will be bolded.
+   */
+  title: PropTypes.string,
+  /**
+   * The type of alert to be rendered. One of Alert.Opts.Types.ALERT, Alert.Opts.Types.ERROR, Alert.Opts.Types.WARNING, Alert.Opts.Types.ADVISORY,
+   * Alert.Opts.Types.INFO, Alert.Opts.Types.SUCCESS, Alert.Opts.Types.GAP_CHECKING, Alert.Opts.Types.OUTSIDE_RECORDS, Alert.Opts.Types.CUSTOM.
+   * Use the Alert.Opts.Types attribute of the Alert component for access to these type strings.
+   */
+  type: PropTypes.oneOf([
+    AlertTypes.ALERT,
+    AlertTypes.ERROR,
+    AlertTypes.WARNING,
+    AlertTypes.ADVISORY,
+    AlertTypes.INFO,
+    AlertTypes.SUCCESS,
+    AlertTypes.GAP_CHECKING,
```

Comment:
@neilpfeiffer Gap Checking terminology is not understood outside of MPages. Can we change this to "Missing Required Fields" Or something like that?

Reply:
Consensus was reached to pull the GAP_CHECKING concept for the time being. This will allow us to move forward. Once a term is agreed upon for this concept, we can add it passively to the component.